web/src/routes/admin/system-settings/+page.svelte

```diff
@@ -68,104 +90,128 @@
       title: $t('admin.metadata_settings'),
       subtitle: $t('admin.metadata_settings_description'),
       key: 'metadata',
+      icon: mdiDatabaseOutline,
     },
     {
       component: LibrarySettings,
       title: $t('admin.library_settings'),
       subtitle: $t('admin.library_settings_description'),
       key: 'external-library',
+      icon: mdiBookshelf,
     },
     {
       component: LoggingSettings,
       title: $t('admin.logging_settings'),
       subtitle: $t('admin.manage_log_settings'),
       key: 'logging',
+      icon: mdiFileDocumentOutline,
     },
     {
       component: MachineLearningSettings,
       title: $t('admin.machine_learning_settings'),
       subtitle: $t('admin.machine_learning_settings_description'),
       key: 'machine-learning',
+      icon: mdiRobotOutline,
     },
     {
       component: MapSettings,
       title: $t('admin.map_gps_settings'),
       subtitle: $t('admin.map_gps_settings_description'),
       key: 'location',
+      icon: mdiMapMarkerOutline,
     },
     {
       component: NotificationSettings,
       title: $t('admin.notification_settings'),
       subtitle: $t('admin.notification_settings_description'),
       key: 'notifications',
+      icon: mdiBellOutline,
     },
     {
       component: ServerSettings,
       title: $t('admin.server_settings'),
       subtitle: $t('admin.server_settings_description'),
       key: 'server',
+      icon: mdiServerOutline,
     },
     {
       component: StorageTemplateSettings,
       title: $t('admin.storage_template_settings'),
       subtitle: $t('admin.storage_template_settings_description'),
       key: 'storage-template',
+      icon: mdiFolderOutline,
     },
     {
       component: ThemeSettings,
       title: $t('admin.theme_settings'),
       subtitle: $t('admin.theme_settings_description'),
       key: 'theme',
+      icon: mdiPaletteOutline,
     },
     {
       component: TrashSettings,
       title: $t('admin.trash_settings'),
       subtitle: $t('admin.trash_settings_description'),
       key: 'trash',
+      icon: mdiTrashCanOutline,
     },
     {
       component: UserSettings,
       title: $t('admin.user_settings'),
       subtitle: $t('admin.user_settings_description'),
       key: 'user-settings',
+      icon: mdiAccountOutline,
     },
     {
       component: NewVersionCheckSettings,
       title: $t('admin.version_check_settings'),
       subtitle: $t('admin.version_check_settings_description'),
       key: 'version-check',
+      icon: mdiUpdate,
     },
     {
       component: FFmpegSettings,
       title: $t('admin.transcoding_settings'),
       subtitle: $t('admin.transcoding_settings_description'),
       key: 'video-transcoding',
+      icon: mdiVideoOutline,
     },
   ];
+
+  let searchQuery = '';
+
+  $: filteredSettings = settings.filter(({ title, subtitle }) => {
+    const query = searchQuery.toLowerCase();
+    return title.toLowerCase().includes(query) || subtitle.toLowerCase().includes(query);
```

Comment:
Can you include the actual config content on here as well? Basically index into it data.configs via the selected component.

Reply:
Can you elaborate?

Reply:
If I use Google oauth and search for "Google" would it return that result? 

If I search for aac and that is the code I'm using for transcoding would it return that section as well? 

I was just mentioning that you could also search against the current config for each second as well as the title and description.